Cargo.toml

```diff
@@ -73,6 +73,7 @@ bench = false
 default = ["multicore"]
 multicore = ["halo2_proofs/multicore"]
 dev-graph = ["halo2_proofs/dev-graph", "image", "plotters"]
+FROST = []
```

Comment:
I think lowercase is the norm for features?

Reply:
apparently the Rust Book says no thing about that but it doesn't portrait any uppercase examples so I will change this